libr/core/cbin.c

```diff
@@ -2249,7 +2249,6 @@ static int bin_sections(RCore *r, int mode, ut64 laddr, int va, ut64 at, const c
 	int fd = -1;
 	bool printHere = false;
 	sections = r_bin_get_sections (r->bin);
-	bool inDebugger = r_config_get_i (r->config, "cfg.debug");
```

Comment:
this depends on an ifdef, pls rebase

Reply:
https://github.com/radare/radare2/pull/11965/files#diff-0ddfa7703dd3261db67bf367b58b64f6R2252